test/remoteCollection/collect.js

```diff
@@ -162,6 +162,87 @@ describe('test/remoteCollection/collect.js >', () => {
       .catch(done);
     });
 
+    it('collecting data from argus', (done) => {
```

Comment:
no test for the "connecting to salesforce" functionality being added in this pr?

Reply:
Check my above comment @iamigo

Reply:
maybe take a look at how the nforce module runs its tests to see if we can leverage something from there? https://github.com/kevinohara80/nforce/tree/master/test

Reply:
oh, that's too bad :(

ok, so we'll have to run manual tests for this until we can figure something out

Reply:
Yes I have opened few issue on wreck and simple-oauth2. If they fix that then we will remove nforce.

Reply:
Can you look at the source code to find the hardcoded url, then mock that?

Reply:
They are trying to hit that URL so I cant moke behalf of them.

Reply:
I don't understand. 
Do we know what the url is?
Do we know what the response from that url will be?
If so, we should be able to mock it, right?

Reply:
I cant mock behalf of them. I am not posting request. Request is done by them. Give me example how to do this. I have no idea how to do this.

Reply:
<img width="578" alt="screen shot 2018-10-25 at 2 46 28 pm" src="https://user-images.githubusercontent.com/1593996/47532328-dbc12e80-d864-11e8-945a-252414061274.png">

I tried this. This thing is not working. So if you have some other idea let me know.

Reply:
<img width="514" alt="screen shot 2018-10-25 at 2 49 00 pm" src="https://user-images.githubusercontent.com/1593996/47532400-2478e780-d865-11e8-87d7-f2b4b892ae1e.png">
I tried this but its not working . SO if you have any other ideas let me know.

Reply:
Let me know this looks good or not. I tried to add test but not sure this is you want @jgraff2 .

Reply:
@harshkothari410 looks good, that's what I was thinking.